modin/engines/base/io/file_reader.py

```diff
@@ -116,6 +116,17 @@ def file_size(cls, f):
         f.seek(cur_pos, os.SEEK_SET)
         return size
 
+    @classmethod
+    def lines_amount(cls, f, from_begin=False):
```

Comment:
This function is unused?

Reply:
yeah, you're right, that method is unused.

removed it in the latest commit